content/docs/reference-events.md

```diff
@@ -61,15 +61,15 @@ function onClick(event) {
 }
 ```
 
-> Note:
+> ध्यान दें:
 >
-> If you want to access the event properties in an asynchronous way, you should call `event.persist()` on the event, which will remove the synthetic event from the pool and allow references to the event to be retained by user code.
+> यदि आप asynchronous तरीके से इवेंट के प्रॉपर्टीज का उपयोग करना चाहते हैं, तो आपको इवेंट पर `event.persist()` को कॉल करना चाहिए, जो पूल से सिंथेटिक इवेंट को हटा देगा और यूजर कोड द्वारा इवेंट के रेफरेन्सेस को बनाये रखेगा।
 
-## Supported Events {#supported-events}
+## समर्थित इवेंट्स {#supported-events}
 
-React normalizes events so that they have consistent properties across different browsers.
+React इवेंट्स को सामान्य करता है ताकि उनके विभिन्न ब्राउज़रों में सुसंगत गुण हों।
```

Comment:
सामान्य could be left as is since the translation changes the meaning

Reply:
normalizes is different than normal?

Reply:
I meant for it to read normalizes, not सामान्य  and not नार्मल . Either in english/hindi works